composer.json

```diff
@@ -6,20 +6,18 @@
   "license": "GPL-2.0-or-later",
   "minimum-stability": "dev",
   "require": {
-    "drupal/layout_paragraphs": "^1.0@beta",
-    "drupal/paragraphs": "^1.11"
+    "drupal/layout_paragraphs": "dev-1.0.x",
+    "drupal/paragraphs": "^1.11",
+    "drupal/pathauto": "^1.8",
+    "drupal/token": "^1.7",
```

Comment:
Do we need to define these here? They look to me like general dependencies. I suspect there is a missing dependencies on localgov_media, which means we are pulling in localgov_core and these would be better defined there.

Reply:
If you download the localgov_campaign module in a stock Drupal site and then try to test Drupal\Tests\localgov_campaigns\Kernel\CampaignPagesTest, the test fails because it can't find the token **contrib** module :(

Reply:
I am fine adding token and pathauto to localgov_core's composer.json.  Please let me know if I should do so now or a separate ticket should cover that.